napari/_app_model/constants/_menus.py

```diff
@@ -17,6 +17,10 @@
 class MenuId(str, Enum):
     """Id representing a menu somewhere in napari."""
 
+    MENUBAR_VIEW = 'napari/view'
```

Comment:
why are things `'napari:window:view:'` above in `CommandId` but `'napari/view'` here? I like having window in there as it signifies to me the top level menubar.

Reply:
i guess `napari/view` seemed even more top level here than `napari/window/view`... and I think my thought process was that commands needn't be associated with a gui, so having `napari:window` in a command id indicates that it's part of the gui... but menus are by definition gui objects... so they will always be a part of some window